week1/week1-homework.py

```diff
@@ -0,0 +1,93 @@
+from random import randint
+
+# ex:1 Write a python script to print your name and age
+
+my_name = 'Nayana'
+print(my_name)
+
+my_age = 34
+print(my_age)
+
+my_info = {'name': 'Nayana', 'age': 34}
+
+print(f"My name is {my_info['name']}. I am {my_info['age']} years old")
```

Comment:
Good job!

Just for completeness, f-strings, also work with variables, not only dictionaries.
So the following code is also valid:
```
print(f"My name is {my_name}. I am {my_age} years old")
```